spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/annotation/SleuthAdvisorConfig.java

```diff
@@ -143,19 +148,14 @@ public AnnotationMethodsResolver(Class<? extends Annotation> annotationType) {
 
 		public boolean hasAnnotatedMethods(Class<?> clazz) {
 			final AtomicBoolean found = new AtomicBoolean(false);
-			ReflectionUtils.doWithMethods(clazz,
-					new ReflectionUtils.MethodCallback() {
-						@Override
-						public void doWith(Method method) throws IllegalArgumentException,
-								IllegalAccessException {
-							if (found.get()) {
-								return;
-							}
-							Annotation annotation = AnnotationUtils.findAnnotation(method,
-									SleuthAdvisorConfig.AnnotationMethodsResolver.this.annotationType);
-							if (annotation != null) { found.set(true); }
-						}
-					});
+			ReflectionUtils.doWithMethods(clazz, method -> {
```

Comment:
for next time.. make a comment when you have blocks of code that are just reformatting. Ex in the PR, make a line comment like "This is just reformatting.. same behaviour" Doing so allows reviewers to skip down and ignore it.

Reply:
ok